apps/src/templates/teacherDashboard/TeacherDashboard.jsx

```diff
@@ -128,14 +128,28 @@ function TeacherDashboard({
           path={TeacherDashboardPath.standardsReport}
           component={props => applyV1TeacherDashboardWidth(<StandardsReport />)}
         />
-        {/* Break out of Switch if we have 0 students. Display EmptySection component instead. */}
         {studentCount === 0 && (
           <Route
-            component={props =>
-              applyV1TeacherDashboardWidth(
-                <EmptySection sectionId={sectionId} />
-              )
-            }
+            component={props => (
+              <div className={dashboardStyles.emptyClassroomDiv}>
+                {location.pathname === TeacherDashboardPath.progress && (
+                  <div>
+                    <Heading1>{i18n.progress()}</Heading1>
+                    <EmptySection
+                      className={dashboardStyles.emptyClassroomProgress}
+                      hasStudents={false}
+                    />
+                  </div>
+                )}
+                {location.pathname !== TeacherDashboardPath.progress && (
+                  <EmptySection
+                    className={dashboardStyles.emptyClassroom}
+                    sectionId={sectionId}
+                    hasStudents={false}
```

Comment:
```suggestion
                    hasStudents={false}
                    hasCurriculumAssigned={false}
```
I think I'd prefer to explicitly set both of these each time. Alternatively, we could use an enum object in the EmptySection. e.g. emptyType.NO_STUDENTS vs emptyType.NO_ASSIGNED_CURRICULUM to send in.